DigitalLearningSolutions.Web/ViewModels/Common/SearchablePage/BasePaginatedViewModel.cs

```diff
@@ -27,6 +27,7 @@ protected BasePaginatedViewModel(PaginationResult<T> paginationResult)
             ItemsPerPage = paginationResult.ItemsPerPage;
             MatchingSearchResults = paginationResult.MatchingSearchResults;
             TotalPages = paginationResult.TotalPages;
+            JavascriptSearchSortFilterPaginateEnabled = paginationResult.TotalItems <= 100;

```

Comment:
Just had an update from Kevin that this should be 250 instead of 100.
I feel like it might also make sense to put this in the app config so it's a bit easier to change if HEE want to change it themselves for any reason. It also makes it nice and clear why JS pagination only works at certain times rather than having it hidden away in a view model.